src/NuGetPackageVerifier/CompositeRules/PackageDependenciesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageDependenciesCompositeRule : IPackageVerifierRule
```

Comment:
Isn't it generally problematic if we have RTM packages relying on prerelease packages?